com.github.philip_scott.spice-up.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "com.github.philip_scott.spice-up",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.philip-scott.spice-up",
+    "rename-icon": "com.github.philip-scott.spice-up",
+    "rename-desktop-file": "com.github.philip-scott.spice-up.desktop",
+    "rename-appdata-file": "com.github.philip-scott.spice-up.appdata.xml",
+    "finish-args": [
+            "--share=ipc", "--socket=x11",
+            "--device=all",
+            "--socket=wayland",
+            "--filesystem=home",
+            /* this is bad but doesn't work otherwise */
+            "--socket=session-bus",
+            "--talk-name=ca.desrt.dconf",
+            "--filesystem=xdg-run/dconf",
+            "--filesystem=~/.config/dconf:ro",
+            "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [
+        {
+        "name": "libgee",
+            "build-options": {
+                "env": {
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.0.tar.xz",
+                    "sha256": "21308ba3ed77646dda2e724c0e8d5a2f8d101fb05e078975a532d7887223c2bb"
+                }
+            ]
+        },
+        {
+            "name": "granite",
+            "buildsystem": "cmake",
```

Comment:
Just a general best practice is to try `cmake-ninja` and if it works cool since its slightly faster.